src/app/api/v1/fxa-rp-events/route.ts

```diff
@@ -256,7 +256,7 @@ export async function POST(request: NextRequest) {
 
         const refreshToken = subscriber.fxa_refresh_token ?? "";
         const accessToken = subscriber.fxa_access_token ?? "";
-        if (accessToken === null || refreshToken === null) {
+        if (!accessToken || !refreshToken) {
           logger.error("failed_changing_password", {
             subscriber_id: subscriber.id,
             fxa_refresh_token: subscriber.fxa_refresh_token,
```

Comment:
Q: Should these two lines (sorry, cant seemingly comment on L263 for some GitHub reason) use the `refreshToken` and `accessToken` coalesced variables from above? Or maybe the logger doesn't really care about the potentially nullish values when logging.

Reply:
Fixed, thanks!